packages/runtime/src/state/modules/read-only-documents.ts

```diff
@@ -29,10 +29,6 @@ export type EmbeddedDocument = BaseDocument & {
   id: string
   type: string
   name: string
-  meta?: {
-    isInitialData?: boolean
-    hasFallback?: boolean
-  }
```

Comment:
Not a breaking change since this is optional. Both of these fields are exclusively for fallback related builder logic.